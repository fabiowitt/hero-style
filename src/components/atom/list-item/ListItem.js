import React from 'react';
import PropTypes from 'prop-types';

import css from './ListItem.module.scss';

const ListItem = ({ children }) => {
  return <li className={css['c-list-item']}>{children}</li>;
};

ListItem.propTypes = {
  children: PropTypes.oneOfType([PropTypes.object, PropTypes.array])
};

ListItem.defaultProps = {
  children: null
};

export default ListItem;
