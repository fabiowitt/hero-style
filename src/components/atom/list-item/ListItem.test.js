import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ListItem from './ListItem';

describe('ListItem', () => {
  it('should match with the snapshot', () => {
    const children = <div>Content</div>;
    const wrapper = shallow(<ListItem>{children}</ListItem>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
