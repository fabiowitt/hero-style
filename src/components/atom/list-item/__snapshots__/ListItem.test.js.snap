// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should match with the snapshot 1`] = `
<li
  className="c-list-item"
>
  <div>
    Content
  </div>
</li>
`;
